oneflow/python/test/modules/test_activation.py

```diff
@@ -119,5 +119,169 @@ def test_gelu_v3(test_case):
         test_case.assertTrue(np.allclose(y.numpy(), z, rtol=1e-4, atol=1e-4))
 
 
+@unittest.skipIf(
+    not flow.unittest.env.eager_execution_enabled(),
+    ".numpy() doesn't work in lazy mode",
+)
+class TestSigmoidModule(flow.unittest.TestCase):
+    def test_sigmoid(test_case):
+        m = flow.nn.Sigmoid()
+        x = flow.Tensor(
```

Comment:
这里也改一下吧，sigmoid的numpy实现：

```suggestion
def sigmoid(x):
  return 1 / (1 + math.exp(-x))
```